src/pretalx/api/serializers/room.py

```diff
@@ -23,7 +23,16 @@ def get_url(self, obj):
 
     class Meta:
         model = Room
-        fields = ("id", "guid", "name", "description", "capacity", "position", "url")
+        fields = (
+            "id",
+            "guid",
```

Comment:
I don't think we need `guid` and `uuid`, just `uuid` should be enough, right? We never want `uuid` when `guid` is set, after all, as that would be confusing.

Reply:
It depends on your use case... I explicitly included both so one can debug if the guid was filled in manually or automatically generated – but we can change that if you prefer to only have one.

Reply:
Yes, please. I want this feature, in use for only a very small group of users, to take up as little space and maintenance effort as possible (to the point that I keep regretting that we didn't put it into a plugin.)